src/main/java/com/faforever/moderatorclient/ui/voting/VotingSubjectAddController.java

```diff
@@ -60,21 +60,21 @@ public void onSave() {
         if (!validate()) {
             return;
         }
-        VotingSubjectFX votingSubjectFX = new VotingSubjectFX();
-        votingSubjectFX.setSubjectKey(subjectKeyTextField.getText());
-        votingSubjectFX.setDescriptionKey(descriptionKeyTextFiled.getText());
-        votingSubjectFX.setMinGamesToVote(Integer.parseInt(minGamesTextField.getText()));
-        votingSubjectFX.setTopicUrl(topicUrlTextField.getText());
+        VotingSubject votingSubject = new VotingSubject();
+        votingSubject.setSubjectKey(subjectKeyTextField.getText());
+        votingSubject.setDescriptionKey(descriptionKeyTextFiled.getText());
+        votingSubject.setMinGamesToVote(Integer.parseInt(minGamesTextField.getText()));
+        votingSubject.setTopicUrl(topicUrlTextField.getText());
         if (!beginTimeTextField.getText().isEmpty()) {
             OffsetDateTime beginTime = OffsetDateTime.of(LocalDateTime.parse(beginTimeTextField.getText(), DateTimeFormatter.ISO_LOCAL_DATE_TIME), ZoneOffset.UTC);
-            votingSubjectFX.setBeginOfVoteTime(beginTime);
+            votingSubject.setBeginOfVoteTime(beginTime);
         }
 
         OffsetDateTime endTime = OffsetDateTime.of(LocalDateTime.parse(endTimeTextField.getText(), DateTimeFormatter.ISO_LOCAL_DATE_TIME), ZoneOffset.UTC);
-        votingSubjectFX.setEndOfVoteTime(endTime);
+        votingSubject.setEndOfVoteTime(endTime);
 
         try {
-            if (votingService.create(votingSubjectFX) == null) {
+            if (votingService.create(votingSubject) == null) {
```

Comment:
We are separating the api entities from the fx beans on purpose. There is a strict separation between API dtos and our UI entities. Revert these changes.